src/core/PDFContext.ts

```diff
@@ -56,6 +56,7 @@ class PDFContext {
     ID?: PDFObject;
   };
   rng: SimpleRNG;
+  parseWarnings: string[];
```

Comment:
It should be named `parsingWarnings` if it is about the parsing.

Reply:
Can you explain what is the purpose of storing the warnings, as they are already outputted on the console by default ?

Reply:
It might be nice to know if the returned pdf had some error/warnings, so it can associate the pdf file/process with the warnings. Currently the console only output the warning with the object ref, which makes it hard to debug.